tilingManager/tilingLayouts/custom/ratio.js

```diff
@@ -4,12 +4,14 @@ const {
     BaseGrabbableLayout
 } = Me.imports.tilingManager.tilingLayouts.custom.baseGrabbable;
 
+let ratio = null;
```

Comment:
this is only going to work if `RatioLayout` is always used as a singleton.

Suggest saving `ratio` as a member variable (that gets initialized to `global.tilingManager.ratio` in the constructor).